### DISCUSSION TITLE
Is OmniGibson support WSL2?

### DISCUSSION THREAD

**Discussion by codetea-fly** [StanfordVL/BEHAVIOR-1K#1011] [Q&A] [answered]
Hi, i tried to run OmniGibson in WSL2. it will crash when isaacsim startup. Is OmniGibson or isaacsim support WSL2?
I have installed vulkan and updated the mesa.

**Comment by cgokmen** [comment, marked as answer]
I don't think Isaac Sim supports WSL, but it does support Windows directly (e.g. you can just install in a Windows conda environment)

**Reply by codetea-fly** [reply]
thank you for reply! i will use windows conda env instead.
